opimodel/colors.py

```diff
@@ -1,24 +1,34 @@
+from opimodel import utils
+import sys
+
+
 class Color(object):
 
-    def __init__(self, r, g, b, name=None):
-        self.red = r
-        self.green = g
-        self.blue = b
+    def __init__(self, rgb=(0, 0, 0),  name=None):
+        self.red, self.green, self.blue = rgb
         self.name = name
 
 
-def parse_color_file(filename):
-    def_colors = {}
-    with open(filename) as f:
+def parse_css_color_file(filepath):
+    """
+    Parse the provided color.def file, create Color objects for each
+    defined color and attach them to the namespace of this module wth
+    names converted into appropriate constants by the
+    utils.mangle_name() function.
```

Comment:
It would be more helpful to describe what the function is doing not just referencing the function. 
Saying 'appropriate constants' means I have to go and find/read the `utils.mangle_name` docs. Also, if you ever refactor the code that comment will be out-of-date.